hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ContainerReplicaPendingOps.java

```diff
@@ -108,7 +113,11 @@ public void scheduleDeleteReplica(ContainerID containerID,
    */
   public boolean completeAddReplica(ContainerID containerID,
       DatanodeDetails target, int replicaIndex) {
-    return completeOp(ADD, containerID, target, replicaIndex);
+    boolean completed = completeOp(ADD, containerID, target, replicaIndex);
+    if (isMetricsNotNull() && completed && replicaIndex > 0) {
```

Comment:
just for myself learning~ what does it mean if replicaIndex == 0?

Reply:
Replicas with an index of zero are Ratis. Indexes greater than zero indicate Erasure Coding replicas.

Reply:
Thanks Stephen!!